DESCRIPTION

```diff
@@ -55,7 +55,8 @@ Suggests:
     styler (>= 1.2.0),
     testthat (>= 2.1.0)
 Remotes: 
-    r-lib/gert
+    r-lib/gert,
+    r-lib/cpp11
```

Comment:
usethis itself does not Import cpp11, nor does it need to, I think.

This is relevant to the package `use_cpp11()` is acting on, yeah?

Reply:
Sorry this was left over from running testing this locally, I will remove it.

Reply:
Actually it is for usethis and cpp11 itself, `desc::desc(package = "cpp11")` needs a installed version of cpp11, so I should add cpp11 to Suggests I guess.

Reply:
Well, this is guarded by `use_dependency("cpp11", "LinkingTo")` which will error if cpp11 is not installed. We have a lot of precedence for this with, e.g., Rcpp, RcppArmadillo, data.table, and more. Otherwise, any package that usethis helps to setup becomes a dependency of usethis itself.

Reply:
Am I missing how cpp11 can't be handled in a similar way?

Reply:
Sure, it can, but how are you going to run the tests if it is not in suggests?

Reply:
Oh I guess we are using a mock in the tests, I can do the same for this.